airbyte-webapp/src/pages/connectorBuilder/ConnectorBuilderLandingPage/ConnectorBuilderLandingPage.tsx

```diff
@@ -30,10 +30,10 @@ import {
   useConnectorBuilderLocalStorage,
 } from "services/connectorBuilder/ConnectorBuilderLocalStorageService";
 
+import { ReactComponent as AirbyteLogo } from "./airbyte-logo.svg";
```

Comment:
I moved these SVGs from the `public` folder into this folder, because vite was giving me an error saying that javascript could not import images from `/public`. These are only used here anyway, so it seemed fine to me

Reply:
This is also the correct implementation: https://github.com/airbytehq/airbyte/issues/15956

Whenever you touch something with an asset still in `public` please feel free to move it directly into the approprivate components.